bundles/multitenant-managed-api-service/1.21.0/bundle.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=multitenant-managed-api-service
```

Comment:
did we confirm that the package name of the existing install is multitenant-managed-api-service?

Reply:
It seems that existing package name is : ```"packageName":"managed-api-service","version":"1.20.0"}```

Reply:
Currently installed mt RHOAM is called managed-api-service.v1.20.0. The upgrade to this version will fail because of it